python/pyspark/pandas/namespace.py

```diff
@@ -2822,6 +2823,8 @@ def broadcast(obj: DataFrame) -> DataFrame:
     """
     Marks a DataFrame as small enough for use in broadcast joins.
 
+    .. deprecated:: 3.2.0
+        Use :func:`DataFrame.spark.hint` instead.
```

Comment:
@itholic can you make a followup that adds a newline here? Sphinx build fails with:

```bash
/.../spark/python/pyspark/pandas/namespace.py:docstring of pyspark.pandas.broadcast:6: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/.../spark/python/pyspark/pandas/namespace.py:docstring of pyspark.pandas.broadcast:7: WARNING: Unexpected section title.
```

I don't know why it passes in GItHub Actions though.

Reply:
Just created a follow-up PR at https://github.com/apache/spark/pull/33479